packages/crawler/src/helpers/fs-helper.ts

```diff
@@ -0,0 +1,55 @@
+import { crawlerConfig as config } from '../../constants';
+import fs from 'fs';
+
+const getWebsiteNameByWebsiteUrl = (websiteUrl: string): string => {
```

Comment:
Return type is missing, too.

Reply:
I guess you meant "writeWebsiteReport" function - fixed it there too